configs/jovo.json

```diff
@@ -2,29 +2,52 @@
   "index_name": "jovo",
   "start_urls": [
     {
-      "url": "https://www.jovo.tech/docs/reference/",
+      "url": "https://v4.jovo.tech/docs/reference/",
```

Comment:
Our v4 page doesn't hav a reference. Actually, our v3 one doesn't have one either, could `reference` be removed completely?

Reply:
Done!